apps/climatemappedafrica/src/lib/data/common/index.js

```diff
@@ -47,17 +47,39 @@ function getNavBar(siteSettings, variant) {
   return {
     logo: imageFromMedia(title, primaryLogo.url),
     drawerLogo: imageFromMedia(title, drawerLogo.url),
+    explorePageUrl: slug,
     menus,
     socialLinks,
     variant,
   };
 }
 
+async function processExplorePage(slugs, hurumap, explorePage) {
```

Comment:
Why is this being handled different from other blocks?

Reply:
@kilemensi Similar to the above, based on the conversation [here](https://github.com/CodeForAfrica/ui/pull/945#issuecomment-2410564857), the explore page can never have any other block/component other than the interactive map. Also, we don't expect the map to be used on more than 1 page, so we don't need to turn it into a block.